docs/book/v3/standard-filters.md

```diff
@@ -1156,11 +1156,11 @@ $filtered = $filter->filter($path);
 ### Non-Existing Paths
 
 Sometimes it is useful to get paths to files that do n0t exist; e.g., when you
-want to get the real path for a path you want to create. You can then either
-provide a `FALSE` `exists` value at initiation, or use `setExists()` to set it.
+want to get the real path for a path you want to create. You can then
+provide a `FALSE` `exists` option at initiation.
```

Comment:
> # Writing Style
> 
> ## One Sentence per-line Method
> 
> Documentation should be written using [the one sentence per-line method](https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line). 
> There are a number of excellent reasons for writing in this style, including:
> 
> - It prevents reflows (meaning a change early in the paragraph won’t cause the remaining lines in the paragraph to reposition)
> - You can easily swap/move sentences
> - You can easily separate or join paragraphs
> - You can comment out sentences or add commentary to them
> - You can spot sentences which are too long or sentences that vary widely in length
> - You can spot redundant (and thus mundane) patterns in your writing
> 
> Take the following content as an example.
> 
> ```markdown
> The concrete `TableGateway` object uses constructor injection for getting
> dependencies and options into the instance. The table name and an instance of
> an `Adapter` are all that is required to create an instance. Out of the box, this 
> implementation makes no assumptions about table structure or metadata, and 
> when `select()` is executed, a simple `ResultSet` object with the populated 
> `Adapter`'s `Result` (the datasource) will be returned and ready for iteration.
> ```
> 
> While it can be easily read in any editor, as it does not require the reader to scroll, that simplicity comes at the cost of maintainability. 
> For example, if you needed to put the second sentence before the first, or move it to any other location, it would need to be cut out of the surrounding paragraph before it could be moved.
> 
> Compare that with the following sample content.
> 
> ```markdown
> The concrete `TableGateway` object uses constructor injection for getting dependencies and options into the instance. 
> The table name and an instance of an `Adapter` are all that is required to create an instance.
> Out of the box, this implementation makes no assumptions about table structure or metadata, and when `select()` is executed, a simple `ResultSet` object with the populated `Adapter`'s `Result` (the datasource) will be returned and ready for iteration.
> ```
> 
> Yes, the content, by default scrolls out of view. 
> However, most text editors and IDEs support "_soft wrapping_", making this a non-issue. 
> Using this approach, the sentence doesn't need to be extracted from the surrounding paragraph before it can be moved.

Reply:
Sorry Frank! the _documented_ style guide… Where is it?

Reply:
Will do. Thanks both